app/resources/migrations/20240411093448_initial_migration.php

```diff
@@ -36,5 +36,14 @@ public function change(): void
         ->addTimestamps(null, false) # https://book.cakephp.org/phinx/0/en/migrations.html#valid-column-options
         ->addForeignKey('id_customer', 'customers', 'id', ['delete'=> 'CASCADE', 'update'=> 'CASCADE'])
         ->create();
+
+        $table = $this->table('documents');
+        $table->addColumn('id_folder', 'integer')
+        ->addColumn('title', 'string', ['limit' => 100])
+        ->addColumn('description', 'string', ['limit' => 500])
+        ->addColumn('language', 'string', ['limit' => 64])
```

Comment:
Das passt so, aber in Zukunft müssen wir eventuell mit ENUM oder einer weitere Tabelle languages arbeiten.